docs/reference/commandline/images.md

```diff
@@ -52,6 +52,35 @@ uses up the `VIRTUAL SIZE` listed only once.
     postgres                  9.3.5               746b819f315e        4 days ago          213.4 MB
     postgres                  latest              746b819f315e        4 days ago          213.4 MB
 
+### Listing images by name and tag
+
+The ``docker images`` command takes an optional ``[REPOSITORY[:TAG]]``
+argument that will filter the list of images for the given
+repository.
+
+    $ docker images java
+    REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+    java                8                   308e519aac60        6 days ago          824.5 MB
+    java                7                   493d82594c15        3 months ago        656.3 MB
+    java                latest              2711b1d6f3aa        5 months ago        603.9 MB
+
+If the ``REPOSITORY`` is specified without any tags, this will list all
+the images that share the same repository with different tags. When a ``TAG``
+is present, this will list only the repository that match with the given tag.
+If nothing matches ``REPOSITORY[:TAG]``, the list will be empty.
+
+    $ docker images java:8
+    REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+    java                8                   308e519aac60        6 days ago          824.5 MB
+    $ docker images java:7
+    REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+    java                7                   493d82594c15        3 months ago        656.3 MB
+    $ docker images java:0 # There is no such image with this tag
+    REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+
+> NOTE: the lookup on ``REPOSITORY`` is done on the entire word. This means
+> ``docker images jav`` won't match the image ``java``.
+
```

Comment:
It's already better than mine so I'll take it :+1: :stuck_out_tongue_closed_eyes: